iree/compiler/Translation/IREEVM.cpp

```diff
@@ -170,6 +171,7 @@ static void buildIREEVMTransformPassPipeline(
 
   switch (inputOptions.type) {
     case InputDialectOptions::Type::none:
+      buildCommonInputConversionPassPipeline(passManager);
```

Comment:
Maybe this belongs out of the switch-case (replace with just a break) and just before the FlowTransformPipeline. Essentially input could be a mix of TOSA/MHLO and Standard Dialect. The TOSA/MHLO are lowered first, then Standard Dialect is.

Reply:
Ok, I debated doing it that way. Let me pop it out of the TOSA/MHLO lowerings and put it here.